src/pychart/__main__.py

```diff
@@ -0,0 +1,18 @@
+import argparse
+from .runner import run_prompt, run_file
+
+def main():
+    parser = argparse.ArgumentParser(prog='pychart')
+    parser.add_argument('--run', '-r', help='run a pychart file')
```

Comment:
What about a non `--run` option as well.

`pytest myfile.pych` should run `myfile` imo,
ofc we can leave what u have currently just add an extra rule